pilco/utils.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
```

Comment:
These are only used in the examples, right? If so, we should move them in the examples folder.